Frameworks/include/UIPopoverPresentationControllerInternal.h

```diff
@@ -16,14 +16,11 @@
 
 #pragma once
 
-@interface UIPopoverPresentationController () {
-@public
-    BOOL _managesPresentation;
-    dispatch_block_t _willDismissCompletion;
-}
+@interface UIPopoverPresentationController()
```

Comment:
I need this header for the class extension / anonymous category exposing methods for UIViewController. I did fix a whitespace error with the "()" in the last commit.